script.js

```diff
@@ -129,4 +129,5 @@ $(document).ready(function() {
     var isExpanded = this.getAttribute("aria-expanded") === "true";
     this.setAttribute("aria-expanded", !isExpanded);
   });
+
```

Comment:
remove blank line? this file should have no changes

Reply:
The blank line is still showing here. you can view the diff on your PR here https://github.com/trailbehind/NG_Help_Center/pull/6/files